src/portals/hope/features/claims/claim-details/ClaimSubclaims/components/SubclaimType/SubclaimType.tsx

```diff
@@ -89,40 +30,24 @@ gql`
       propertyId: $propertyId
       optionIds: $optionIds
     ) {
-      id
-      type
-      propertySelections {
-        claimType
-        property {
-          id
-          name
-        }
-        option {
-          id
-          name
-        }
-      }
+      ...ClaimInformationSubclaim
     }
   }
 `
 
 export const SubclaimType: React.FC<{ subclaimId: string }> = ({
   subclaimId,
 }) => {
-  const { preferredCurrency } = useClaim()
-  const { data: typeInformation } = useSubclaimTypeInformationQuery({
-    variables: { subclaimId },
-  })
-
+  const { preferredCurrency, getSubclaim } = useClaim()
   const [setClaimType] = useSetSubclaimTypeMutation()
+  const subclaim = getSubclaim(subclaimId) as ClaimInformationSubclaimFragment
```

Comment:
I guess we can speicify this return type?
```suggestion
  const subclaim = getSubclaim(subclaimId)
```

Reply:
It is actually specified... but it is ClaimInformationSubclaimFragment | undefined, which the optimisticResponse didn't like 🤪